client/src/routes/Temple/components/Counter/Counter.tsx

```diff
@@ -17,12 +17,16 @@ const Counter: React.FC<CounterProps> = ({startTime, starting = false}) => {
   const [now, setNow] = useState(dayjs());
 
   useEffect(() => {
+    if (dayjs.duration(startTime.diff(now)).days() > 0) {
+      return;
+    }
+
     const interval = setInterval(() => {
       setNow(dayjs());
     }, 1000);
 
     return () => clearInterval(interval);
-  }, []);
+  }, [now, startTime]);
```

Comment:
Will this not recreate the intervall for each now? `setNow` will cause it to run again and so it goes

Reply:
true! let me have a look

Reply:
should be fixed now, nice catch